supplier_app/tests_company.py

```diff
@@ -0,0 +1,122 @@
+from django.test import (
+    Client,
+    TestCase,
+)
+
+from django.urls import (
+    reverse_lazy
+)
+
+from supplier_app.models import (
+    Company,
+    TaxPayer,
+    CompanyUserPermission,
+)
+from supplier_app.views import (
+    CompanySelectorView,
+)
+from users_app.models import User
+
+TAXPAYER = {
+    'name': 'Eventbrite',
+    'workday_id': '12345',
+}
+COMPANY = {
+    'name': 'Eventbrite',
+    'description': 'Bringing the world together through live experiences',
+}
+USER = {
+    'email': 'pepe@gmail.com',
+    'password': '123456'
+}
+
+
+class TestCompanyModels(TestCase):
+
+    def test_company(self):
+        company = Company.objects.create(**COMPANY)
+        self.assertEqual(company.name, COMPANY['name'])
+        self.assertEqual(company.description, COMPANY['description'])
+        self.assertEqual(
+            str(company),
+            "{}".format(
+                COMPANY['name']
+            )
```

Comment:
Are you sure about it? `name` is already an string.

Reply:
you are right